dusty/compiler/spec_assembler.py

```diff
@@ -140,15 +144,27 @@ def get_repo_of_app_or_library(app_or_library_name):
         return Repo(specs['libs'][app_or_library_name]['repo'])
     raise KeyError('did not find app or service with name {}'.format(app_or_library_name))
 
+def _get_respective_schema(specs_type):
+    if specs_type == 'apps':
+        return app_schema
+    elif specs_type == constants.CONFIG_BUNDLES_KEY:
+        return bundle_schema
+    elif specs_type == 'libs':
+        return lib_schema
```

Comment:
Guessing this is supposed to return `None` if you give it `services`. Should probably raise if you give it anything else, the bit a few lines down where you dispatch on the truthiness of this seems a bit risky otherwise.